integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/DistroXClusterFaultTolerantUpscaleTest.java

```diff
@@ -0,0 +1,223 @@
+import org.testng.annotations.Test;
+
+import com.cloudera.api.swagger.model.ApiHost;
+import com.cloudera.api.swagger.model.ApiHostList;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.common.Status;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.base.InstanceStatus;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.instancegroup.InstanceGroupV4Response;
+import com.sequenceiq.cloudbreak.common.json.JsonUtil;
+import com.sequenceiq.cloudbreak.orchestrator.salt.domain.MinionIpAddressesResponse;
+import com.sequenceiq.it.cloudbreak.client.BlueprintTestClient;
+import com.sequenceiq.it.cloudbreak.client.DistroXTestClient;
+import com.sequenceiq.it.cloudbreak.client.SdxTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.MockedTestContext;
+import com.sequenceiq.it.cloudbreak.context.RunningParameter;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.dto.distrox.cluster.DistroXClusterTestDto;
+import com.sequenceiq.it.cloudbreak.dto.distrox.cluster.clouderamanager.DistroXClouderaManagerTestDto;
+import com.sequenceiq.it.cloudbreak.dto.distrox.image.DistroXImageTestDto;
+import com.sequenceiq.it.cloudbreak.dto.distrox.instancegroup.DistroXNetworkTestDto;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+import com.sequenceiq.it.cloudbreak.testcase.mock.clouderamanager.AbstractClouderaManagerTest;
+
+public class DistroXClusterFaultTolerantUpscaleTest extends AbstractClouderaManagerTest {
+
+    private static final int SUCCESSFUL_UPSCALE_NODE_COUNT = 10;
+
+    private static final int FAILING_NODE_COUNT = 2;
+
+    private static final String UPSCALED_INSTANCE_GROUP_NAME = "worker";
+
+    private static final String IMAGE_CATALOG_ID = "f6e778fc-7f17-4535-9021-515351df3691";
+
+    private static final String CM_FOR_DISTRO_X = "cm4dstrx";
+
+    private static final String CLUSTER_KEY = "cmdistrox";
+
+    private static final String DIX_IMG_KEY = "dixImg";
+
+    private static final String DIX_NET_KEY = "dixNet";
+
+    private static final Duration POLLING_INTERVAL = Duration.of(3000, ChronoUnit.MILLIS);
+
+    @Inject
+    private BlueprintTestClient blueprintTestClient;
+
+    @Inject
+    private DistroXTestClient distroXClient;
+
+    @Inject
+    private SdxTestClient sdxTestClient;
+
+    @Test(dataProvider = TEST_CONTEXT_WITH_MOCK)
+    @Description(
+            given = "there is a running DistroX cluster",
+            when = "an upscale is called, but some of the new nodes are unreachable",
+            then = "the cluster should be AVAILABLE in a partially upscaled state")
+    public void testUpscaleWithFewNodesBeingUnreachable(MockedTestContext testContext, ITestContext testNgContext) {
+        DistroXStartStopTestParameters params = new DistroXStartStopTestParameters(testNgContext.getCurrentXmlTest().getAllParameters());
```

Comment:
I think this is not necessary if the `UPSCALED_INSTANCE_GROUP_NAME = "worker"` defined here. So we can remove the reference (`DistroXStartStopTestParameters params`) and use `UPSCALED_INSTANCE_GROUP_NAME` everywhere in this class.

Reply:
Removed unnecessary params variable.